changelog.d/12541.misc

```diff
@@ -0,0 +1 @@
+Define Dockerfile version to fix issues with building images in some GitLab CI environments due to Buildkite.
```

Comment:
We've got a changelog section for the docker image. Can you rename this file to `12541.docker`?

Reply:
```suggestion
Explicitly opt-in to using [BuildKit-specific features](https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md) in the Dockerfile. This fixes issues with building images in some GitLab CI environments.
```